autogpt_platform/backend/backend/blocks/embeddings/jina.py

```diff
@@ -0,0 +1,44 @@
+from typing import Literal
+
+import requests
+
+from backend.data.block import Block, BlockCategory, BlockOutput, BlockSchema
+from backend.data.model import BlockSecret, SchemaField, SecretField, CredentialsMetaInput, CredentialsField
+
+
+class JinaEmbeddingBlock(Block):
+    class Input(BlockSchema):
+        texts: list = SchemaField(description="List of texts to embed")
+        credentials: CredentialsMetaInput[Literal['jina'], Literal['api_key']] = CredentialsField(
+            provider="jina",
+            supported_credential_types={"api_key"}, # noqa
```

Comment:
You shouldn’t need the noqa what’s the issue you’re getting? These type hints are used to decide what loads in the system

Reply:
```suggestion
            supported_credential_types={"api_key"},
```